Ruby/fibonacci.rb

```diff
@@ -0,0 +1,20 @@
+def fibonacci(n)
+    a = 0
+    b = 1
+
+    # Compute Fibonacci number in the desired position.
+    n.times do
+        temp = a
+        a = b
+        # Add up previous two numbers in sequence.
+        b = temp + b
+    end
+
+    return a
+end
+
+# Write first 15 Fibonacci numbers in sequence.
+15.times do |n|
```

Comment:
can we not hardcode this and get input from user?

Reply:
comment addressed